tools/cli/pkg/command/reconciliation_info.go

```diff
@@ -140,8 +142,27 @@ func (cmd *ReconciliationOperationInfoCommand) Run() error {
 		return err
 	}
 
-	var result mothership.ReconcilationOperationsOKResponse
-	if err := json.NewDecoder(response.Body).Decode(&result); err != nil {
+	var result ReconcilerInfoResponses
```

Comment:
Split this code to 3 phases:
1 - init client
2 - get reconciliation info
3- get cluster config

This will help to avoid double closing the same response and not closing the first response by defer.

Reply:
done